### ISSUE TITLE
When use mobilenet and CIFAR-10 dataset, mobile can not be trained

### ISSUE THREAD

**Issue by fangliu518** [4uiiurz1/keras-arcface#21]
Could you please look at my code, if I am using the model correctly?
y = Input(shape=(10,))

mobile_V2 = MobileNet(input_shape =(32,32,3), include_top = False, alpha = 0.5)
input = mobile_V2.input

x = BatchNormalization()(mobile_V2.output)

x = Dropout(rate=0.5)(x)
x = Flatten()(x)

x = Dense(10)(x)
x = BatchNormalization()(x)
print(x.shape)
output = ArcFace(10,regularizer=regularizers.l2(weight_decay))([x,y])
print(output.shape)

model = Model([input,y],output)
